templates/aboutUsSection.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="container1">

```

Comment:
I would suggest having a <section> wrapping all the content of this component.
Also you can improve naming on the classes. For example 

```
<section class="aboutus">
    <div class="aboutus__heading">
       <h2 class="about_us__eyebrow">
... and so on
```